src/extensions/default/JavaScriptRefactoring/RenameIdentifier.js

```diff
@@ -100,11 +100,27 @@ define(function (require, exports, module) {
          * If yes then select all references
          */
         function handleFindRefs (refsResp) {
+            var inlineWidget = EditorManager.getFocusedInlineWidget(),
+                editor = EditorManager.getActiveEditor();
+            //In case of inline widget if some references are outside widget's text range then don't allow for rename
+            if (inlineWidget) {
+                var isInTextRange = true;
+                refsResp.references.refs.forEach(function(item) {
```

Comment:
`refsResp.references.refs` might throw. In #L119 it's handled by checking for `if (resResp && ...)`.

I think easiest thing would be null checking it in the start of the function and either returning immediately if it's null and/or saving it to a variable if it is.

```js
function handleFindRefs (refsResp) {
  if (refsResp && refsResp.references && refsResp.references.refs) {
     return;
  }

  var refs = refsResp.references.refs;
  var type = refsResp.references.type;

  ...
}```